src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -206,6 +208,19 @@ export class RoslynLanguageServer {
             );
         });
 
+        // Retrieve the dotnet info outside of the notification so we're not running dotnet --info every time the project changes.
+        const dotnetInfo = await getDotnetInfo([]);
+        this._languageClient.onNotification(RoslynProtocol.ProjectConfigurationNotification.type, (params) => {
+            reportProjectConfigurationEvent(
+                this.telemetryReporter,
+                params,
+                this.platformInfo,
+                dotnetInfo,
+                this._solutionFile?.fsPath,
+                true
+            );
+        });
```

Comment:
I agree we don't want to be running it all the time, but can we just run it the first time we get a message async so we're not holding up the LSP server launch? Or just include the version number in the message itself?